internal/syscall/syscall_nonlinux.go

```diff
@@ -26,15 +26,13 @@ import (
 	"net"
 	"sync"
 	"time"
-
-	"google.golang.org/grpc/grpclog"
 )
 
 var once sync.Once
 
 func log() {
 	once.Do(func() {
-		grpclog.Info("CPU time info is unavailable on non-linux or appengine environment.")
+		logger.Info("CPU time info is unavailable on non-linux or appengine environment.")
```

Comment:
This doesn't compile, right?
logger is only defined for linux, and this file compiles for non-linux.